serverless.yml

```diff
@@ -1,12 +1,16 @@
 # For full config options, check the docs:
 #    docs.serverless.com
 
-service: projectworks-leave-notifications
+# the service name is used as a prefix for infrastructure created via
+# CloudFormation. Some infrastructure has limits on name length so don't make
+# the service name too long
+service: projectworks-leave-notify
 frameworkVersion: '2'
 provider:
   name: aws
   region: ap-southeast-2
   runtime: nodejs12.x
```

Comment:
we should bump this

```suggestion
  runtime: nodejs16.x
```

Reply:
@G-Rath Node 16 isn't a supported thing in serverless yet (I tried to make this change). They support 14.x. I decided to leave this upgrade to later PR

Reply:
yeah its cause we're not on v3 - fwiw that should be a pretty easy upgrade; iirc I didn't have to change anything and it had no impact on the actual lambdas (aside from config adjustments for the deprecated stuff that got removed)